libraries/chain/include/graphene/chain/protocol/custom_account_authority.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+#include <graphene/chain/protocol/base.hpp>
+
+namespace graphene
+{
+namespace chain
+{
+
+struct custom_account_authority_create_operation : public base_operation
+{
+   struct fee_parameters_type
+   {
+      uint64_t fee = 0;
+   };
+
+   asset fee;
+   custom_permission_id_type permission_id;
+   int operation_type;
```

Comment:
I guess this is for storing the operation id. In that case, shouldn't this be of uint32 type since it can't be negative?

Reply:
operation is of type fc::static_variant.
static_variant.which() ==> _tag is of type int.
But https://github.com/peerplays-network/peerplays/pull/365/files#diff-26ab0c872c3ba1e230ea017a0db1361aR15  validates anything coming in.